Library/PngLib/pnglib.gp

```diff
@@ -32,7 +33,10 @@ export PNGIMPORTCLEANUPIDATPROCESSINGSTATE
 export PNGCALCBYTESPERROW
 export PNGCALCBYTESPERPIXEL
 export PNGCALCLINEALLOCSIZE
-export PNGREMOVEALPHACHANNEL
+export PNGALPHACHANNELBLEND
```

Comment:
Adding exports in between, changing export contracts breakes versiokned compatibility. New ones should be added to the end, removed ones when needed should be empty (skipped).

Use incminor before new added exports to signal an compatibility relevant addition